jobs/sql/extractions/all_encounters.sql

```diff
@@ -0,0 +1,153 @@
+SET @partition = '${partitionNum}';
+set sql_safe_updates = 0;
+set @next_appt_date_concept_id = CONCEPT_FROM_MAPPING('PIH', 5096);
+
+drop temporary table if exists temp_all_encounters;
+create temporary table temp_all_encounters
+(
+encounter_id        int,
+encounter_datetime	datetime, 
+encounter_type      int,
+location_id         int,
+patient_id          int, 
+visit_id            int, 
+creator             int(11),
+user_entered        varchar(255),
+encounter_location  varchar(150),
+encounter_type_name varchar(150),
+entered_datetime    datetime,
+emr_id              varchar(15),
+next_appt_date      date,
+index_asc           int,
+index_desc          int		
+);
+
+Insert into temp_all_encounters (
+encounter_id, encounter_datetime, encounter_type, location_id, patient_id, visit_id, creator, entered_datetime)
+select encounter_id, encounter_datetime, encounter_type, location_id, patient_id, visit_id, creator, date_created 
+from encounter 
+where voided = 0
```

Comment:
yeah, there's a handful of them (maybe from manual scripts?).  But I added a subsequent delete statement that deleted rows for voided patients, rather than potentially slow down the insert statement.  The delete statement runs quickly.